src/main/java/org/fundacion/jala/converter/models/UserSQL.java

```diff
@@ -17,6 +17,9 @@
 import javax.persistence.Persistence;
 import java.util.List;
 
+/**
+ * This class is to query to database.
+ */
 public class UserSQL {
     public UserSQL() {
     }
```

Comment:
Change the place of constructor, private static EntityManagerFactory must be above the constructor

Reply:
ok 

it was solved